tests/plugins/test_tag_and_push.py

```diff
@@ -83,14 +83,14 @@
 
 
 class Y(object):
-    pass
+    def __init__(self):
+        self.dockerfile_path = None
+        self.path = None
```

Comment:
Alternatively, we could use `stubs.StubSource` instead of `Y`, and `stubs.StubInsideBuilder` instead of `X`.